### ISSUE TITLE
ReferenceError: dateString is not defined

### ISSUE THREAD

**Issue by mikeevstropov** [GeoffZhu/vue-event-calendar#42]
My template based on example from README.md
```html
<template>

    <div class="fast-navigation-calendar">
        <vue-event-calendar :events="demoEvents"></vue-event-calendar>
    </div>

</template>

<script>

    export default {
        name: 'fast-navigation-calendar',
        data () {
            return {
                demoEvents: [{
                    date: '2016/12/15',
                    title: 'Foo',
                    desc: 'longlonglong description'
                },{
                    date: '2016/11/12',
                    title: 'Bar'
                }]
            }
        }
    }

</script>
```

But i see
```console
[Vue warn]: Error in getter for watcher "calendarParams": "ReferenceError: dateString is not defined"

found in

---> <VueEventCalendar>
       <FastNavigationCalendar> at src/components/fast-navigation/fast-navigation-calendar.vue
         <FastNavigation> at src/components/fast-navigation/fast-navigation.vue
           <FrontendIndex> at src/pages/frontend-index.vue
             <App> at src/app.vue
               <Root>
error during render : /
ReferenceError: dateString is not defined
    at VueComponent.calendarParams (/var/git/frontend/node_modules/vue-event-calendar/dist/index.js:864:26)
    at Watcher.get (/var/git/frontend/node_modules/vue/dist/vue.runtime.common.js:2860:25)
    at Watcher.evaluate (/var/git/frontend/node_modules/vue/dist/vue.runtime.common.js:2967:21)
    at VueComponent.computedGetter (/var/git/frontend/node_modules/vue/dist/vue.runtime.common.js:3238:17)
    at VueComponent.<anonymous> (/var/git/frontend/node_modules/vue/dist/vue.runtime.common.js:432:16)
    at Watcher.get (/var/git/frontend/node_modules/vue/dist/vue.runtime.common.js:2860:25)
    at new Watcher (/var/git/frontend/node_modules/vue/dist/vue.runtime.common.js:2849:12)
    at VueComponent.Vue.$watch (/var/git/frontend/node_modules/vue/dist/vue.runtime.common.js:3338:19)
    at createWatcher (/var/git/frontend/node_modules/vue/dist/vue.runtime.common.js:3298:13)
    at initWatch (/var/git/frontend/node_modules/vue/dist/vue.runtime.common.js:3280:7)
    at initState (/var/git/frontend/node_modules/vue/dist/vue.runtime.common.js:3069:5)
    at VueComponent.Vue._init (/var/git/frontend/node_modules/vue/dist/vue.runtime.common.js:4193:5)
    at new VueComponent (/var/git/frontend/node_modules/vue/dist/vue.runtime.common.js:4365:12)
    at createComponentInstanceForVnode (/var/git/frontend/node_modules/vue-server-renderer/build.js:6810:10)
    at renderComponentInner (/var/git/frontend/node_modules/vue-server-renderer/build.js:6986:40)
    at renderComponent (/var/git/frontend/node_modules/vue-server-renderer/build.js:6961:5)
    at RenderContext.renderNode (/var/git/frontend/node_modules/vue-server-renderer/build.js:6877:5)
    at RenderContext.next (/var/git/frontend/node_modules/vue-server-renderer/build.js:2186:14)
    at cachedWrite (/var/git/frontend/node_modules/vue-server-renderer/build.js:2046:9)
    at renderElement (/var/git/frontend/node_modules/vue-server-renderer/build.js:7101:5)
    at renderNode (/var/git/frontend/node_modules/vue-server-renderer/build.js:6879:5)
    at renderComponentInner (/var/git/frontend/node_modules/vue-server-renderer/build.js:6997:3)
"GET /" 500 - "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/60.0.3112.78 Safari/537.36"
```

Im using
```json
"vue": "^2.4.1",
"vue-event-calendar": "^1.4.5"
```

Exactly
`ReferenceError: dateString is not defined`
https://github.com/GeoffZhu/vue-event-calendar/blob/master/dist/index.js#L864

**Comment by GeoffZhu** [comment]
Fixed in version 1.4.6, try again. @mikeevstropov  :)

**Comment by mikeevstropov** [comment]
@GeoffZhu It WORKS!!!! Thank you so much!!!! You like Flash from DC.
